commerce-inventory-service/src/main/java/com/liferay/commerce/inventory/service/impl/CommerceInventoryWarehouseItemLocalServiceImpl.java

```diff
@@ -57,13 +57,15 @@ public CommerceInventoryWarehouseItem addCommerceInventoryWarehouseItem(
 			externalReferenceCode = null;
 		}
 
-		CommerceInventoryWarehouseItem commerceInventoryWarehouseItem =
-			commerceInventoryWarehouseItemPersistence.findByC_S(
-				commerceInventoryWarehouseId, sku);
+		if (Validator.isNotNull(sku)) {
```

Comment:
What if the `sku` is null/blank here? The db index will still ensure there's only one warehouse item with the given `commerceInventoryWarehouseId, sku` combination.